Pod/Classes/SEGAdobeIntegration.m

```diff
@@ -460,6 +466,41 @@ - (void)trackHeartbeatEvents:(SEGTrackPayload *)payload
         SEGLog(@"[ADBMediaHeartbeat trackEvent:ADBMediaHeartbeatEventBufferStart mediaObject:%@ data:%@]", self.mediaObject, payload.properties);
         return;
     }
+
+    // Video Ad Break Started/Completed are not spec'd. For now, will document for Adobe and
+    // write a proposal to add this to the Video Spec
+    if ([payload.event isEqualToString:@"Video Ad Break Started"]) {
```

Comment:
Think ad events should be added in their own PR. The PR is about initializing Adobe correctly - sending ad events seems like a different task.

Reply:
That change was already merged, I had this against the ad events branch to continue working, so Github just sees that as part of the change